core/src/main/java/com/sequenceiq/cloudbreak/controller/validation/template/TemplateValidatorAndUpdater.java

```diff
@@ -61,6 +62,8 @@ public class TemplateValidatorAndUpdater {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(TemplateValidatorAndUpdater.class);
 
+    private static final Set<String> SDX_COMPUTE_INSTANCES = Set.of("hmshg", "razhg", "atlashg");
```

Comment:
Very dumb request from me, but could you move this above the `LOGGER` just to keep all of the conditional variables in the same section?